doc/interop-test-descriptions.md

```diff
@@ -418,21 +420,32 @@ Procedure:
       }
     }
     ```
+The client attaches custom metadata with the following keys:
+    ```
+    "x-grpc-test-echo-initial"
+    or
```

Comment:
So the client is supposed to issue two calls? Or should the client issue one call with both?

I guess also, what should the values of the keys be?